set.mm

```diff
@@ -579416,7 +579417,7 @@ This section is an experiment to see in practice if (partially) unbundled
   condition is required, we label it bj-xxx.
 
   It is important to keep all the bundled theorems already in set.mm, but one
-  may also add the (partially) unbundled versions which dipense with ~ ax-13 ,
+  may also add the (partially) unbundled versions which depends on ~ ax-13 ,
```

Comment:
I think here "dispense with" is the correct term, see the end of the sentence.

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4867/commits/609c0475942f36cf76851285f530f6a975128e08